### PR TITLE
📖 Add post about joining the CNCF Sandbox

### PR DESCRIPTION
**Pull request by embik**
I drafted a blog post for kcp.io that celebrates joining the CNCF Sandbox. It also gives a brief history of the transition to community governance that happened earlier this year, because that transition has essentially only be documented in a couple of PRs to kcp-dev/kcp.

Please let me know how you feel about the post and its language.

### CODE REVIEW

**Comment by embik** [comment]
/cc @sttts @clubanderson @scheeles @mjudeikis @xrstf 

for review/awareness of all maintainers.

File: content/en/blog/news/2023-10-joining-the-cncf.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+A group of committed organizations and individuals decided to respond to the call for action and established a [new project governance](https://github.com/kcp-dev/kcp/blob/main/GOVERNANCE.md) and bootstrapped a [new list of maintainers](https://github.com/kcp-dev/kcp/pull/2953) to lead kcp as a community project. Since then, more people have joined the list of maintainers to drive the kcp project forward. We relaunched kcp.io in a new design recently to commemorate the "new" kcp project.
+
+From the inception of the new project governance, it was clear that kcp does not exist in a vacuum. It is a citizen of the Kubernetes ecosystem and as such, joining the CNCF to formalize that citizenship was the obvious and natural road to take. Thankfully, after hearing our case, the [Technical Oversight Committee (TOC)](https://www.cncf.io/people/technical-oversight-committee/) decided to accept us into the CNCF Sandbox program.
```
Comment (sttts) [inline content/en/blog/news/2023-10-joining-the-cncf.md]:
```suggestion
From the inception of the new project governance, it was clear that kcp does not exist in a vacuum. It is a citizen of the Kubernetes ecosystem and as such, joining the CNCF to formalize that citizenship was the obvious and natural road to take. Eventually, after evaluating our application, the [Technical Oversight Committee (TOC)](https://www.cncf.io/people/technical-oversight-committee/) decided to accept us into the CNCF Sandbox program.
```

Comment (sttts) [reply content/en/blog/news/2023-10-joining-the-cncf.md]: "Thankfully, after hearing our case" sounds a little like "they saved us" 😀 Might be only in my non-native ears.

**Comment by sttts** [comment]
One nit. Otherwise, great text.

File: content/en/blog/news/2023-10-joining-the-cncf.md (hunk 1 above)
Comment (embik) [reply content/en/blog/news/2023-10-joining-the-cncf.md]: That's fair - "hearing our case" is not the best way to say this, thinking about it.

**Comment by embik** [comment]
/hold

to merge this PR on the date the blog is configured for.

**Comment by embik** [comment]
ping @sttts @clubanderson @scheeles @mjudeikis @xrstf 

This post has been updated as per our last community meeting discussions. Please take a look, the ETA for the post is next Wednesday.

**Comment by mjudeikis** [comment]
looks good!

File: content/en/blog/news/2024-01-joining-the-cncf-sandbox.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Joining the CNCF Sandbox"
+linkTitle: "Joining the CNCF Sandbox"
+date: 2024-01-24
+---
```
Comment (mjudeikis) [inline content/en/blog/news/2024-01-joining-the-cncf-sandbox.md:9]: Does our current engine supports tags for SEO here?

**Review by clubanderson — COMMENTED** [review]
lgtm

File: content/en/blog/news/2024-01-joining-the-cncf-sandbox.md (hunk 2 above)
Comment (embik) [reply content/en/blog/news/2024-01-joining-the-cncf-sandbox.md:9]: Yes! Pushed a commit with a few tags.

**Review by avinal — COMMENTED** [review]
Looks nice to me ❤️

**Comment by embik** [comment]
/approve

since feedback has been positive. Leaving the lgtm up for grabs.

**Comment by sttts** [comment]
Great blog post.

/lgtm
/approve

**Comment by embik** [comment]
/hold cancel

Thank you, folks!

**Comment by embik** [comment]
Post is live at https://www.kcp.io/blog/2024/01/24/joining-the-cncf-sandbox/ 🎉

**Pull request merged**